source/isaaclab/isaaclab/sensors/ray_caster/ray_caster_cfg.py

```diff
@@ -62,6 +61,9 @@ class OffsetCfg:
         * `world` if rays' starting positions and directions are always fixed. This is useful in combination with the grid map package.
     """
 
+    return_distance: bool = False
```

Comment:
Can we call it `track_ray_distance`. This will then match the way we handle different data outputs in ContactSensor class:
https://github.com/isaac-sim/IsaacLab/blob/main/source/isaaclab/isaaclab/sensors/contact_sensor/contact_sensor_cfg.py#L20-L24